### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by helltepakcer**
In this branch we all together fixed some mistakes which we have create while did our features. I have resolved conflicts which were appears, please review and comment if you think that something wrong.

### CODE REVIEW

File: features/check_pingbot.feature [hunk 1]
```diff
@@ -10,4 +10,4 @@ Background:
 Scenario: Check pingbot room
     Given we open pingbot room
     When we send message
-    Then we receive pingbot reply
\ No newline at end of file
+    Then we receive pingbot reply
```
Comment (ghost) [inline features/check_pingbot.feature]: Did u delete line and then added ?)

File: features/environment.py [hunk 2]
```diff
@@ -54,6 +54,7 @@ def before_all(context):
     context.emoticons_page = EmoticonsPage(context)
     context.search_page = SearchPage(context)
     context.people_page = PeoplePage(context)
+    context.test_name = "@gtest"
```
Comment (ghost) [inline features/environment.py]: Do we need dependencies on the test name of "gena" ?

**Review by ghost — COMMENTED** [review]
Commented 2 issues . Waiting for answers

File: features/steps/step_invite_your_team.py [hunk 3]
```diff
@@ -1,26 +1,34 @@
-from behave import when, then
+from behave import when, then, given
 
 
-@when('we click Invite your team')
+@given('we are on Lobby Page')
 def step_impl(context):
-    context.lobby_page.invite_team_form()
+    # context.lobby_page.navigate()
+    pass
```
Comment (Anedchenko) [inline features/steps/step_invite_your_team.py]: Why are we commenting here?

File: pages/people_page.py [hunk 4]
```diff
@@ -9,46 +9,49 @@
 class PeoplePage(Page):
 
     url = '/people'
-    admins = ['Tim Bortnik']
+    admins = ['Tim Bortnik', 'Yevhenii Udovychenko']
```
Comment (Anedchenko) [inline pages/people_page.py:12]: @Bidonus  is it correct solution?

Comment (Uarhodesian) [reply pages/people_page.py:12]: maybe we should to do some function to add (or delete) user to admins list

Comment (helltepakcer) [reply pages/people_page.py:12]: @Uarhodesian, I think you right, let's do it in next split?

File: features/environment.py (hunk 2 above)
Comment (helltepakcer) [reply features/environment.py]: As you can see, I writing my name as 'Hena', but you right) I have change this string .

**Review by ghost — APPROVED** [review]

**Pull request merged**